silverscreen/settings.py

```diff
@@ -113,7 +117,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'EET'
```

Comment:
Any reason why this was chosen?
You can use Europe/Bucharest to view the playing times in our timezone in the admin pages, for example.